source/vstd/arithmetic/power2.rs

```diff
@@ -90,6 +94,27 @@ pub proof fn lemma_pow2_auto()
     }
 }
 
+/// Proof relating 2^e to 2^(e-1) for a specific e.
+pub proof fn lemma_pow2_unfold(e: nat)
```

Comment:
Is "unfold" a good name for this?

I borrowed it from:
https://github.com/verus-lang/verus/blob/1604eeabe5e625b3386733e7649f4914b533f410/source/rust_verify/example/power_of_2.rs#L46

Reply:
This lemma seems very specific.  In the place where it's used, would revealing `pow2` do the trick instead?